config/templates/stack-template.json

```diff
@@ -742,7 +754,13 @@
         "SecurityGroups" : [
           { "Ref" : "SecurityGroupElbAPIServer" }
         ]
-      }
+      },
+      "DependsOn" : [
+        {{range $etcdIndex, $etcdInstance := .EtcdInstances}}
+        {{if gt $etcdIndex 0}},{{end}}
+        "InstanceEtcd{{$etcdIndex}}"
+        {{end}}
+      ]
```

Comment:
@pwillie As redbaron kindly described, it won't happen with latest master or even with v0.9.4-rc.1.
Could you confirm the fix and if you're ok with that, remove this `DependsOn` part?